src/style-spec/bin/gl-style-composite.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-unresolved, no-process-exit */
```

Comment:
Nit: `import/no-unresolved` no longer needed

Reply:
Unfortunately in that case we need to add the `@mapbox/mapbox-gl-style-spec` as a dependency to itself, or silence ESLint with

```
/* eslint import/no-unresolved: [error, { ignore: ['^@mapbox/mapbox-gl-style-spec$'] }] */
```

I've updated the PR to ignore the `@mapbox/mapbox-gl-style-spec` dependency